tests/features/asset_distribution/add_asset_distribution.feature

```diff
@@ -198,5 +198,6 @@ Feature: Add distribution through the UI
       | Quality Yard | Boost Software License | Solution random x name |
 
     When I go to the homepage of the "Solution random x name" solution
-    Then I should not see the text "WTFPL"
-    And I should not see the text "Boost Software License"
+    # Distributions are still shown in the solution page and so the License text will be visible in the page.
```

Comment:
I tested this manually by adding a standalone distribution on the solution and I also queried the production server to verify that this is the current status quo of this functionality.
To query the production server, on production's SPARQL endpoint, you can run the following query to get a list of solutions that have at least one standalone distribution:
```
select DISTINCT(?entity)
where {
  ?entity a <http://www.w3.org/ns/adms#Asset> .
  ?entity <http://www.w3.org/ns/dcat#distribution> ?distr .
  FILTER NOT EXISTS { ?entity <http://purl.org/dc/terms/isVersionOf> ?parent }
}
```
Adding a standalone distribution through the UI also makes this distribution appear in the overview page.
I can assume that the previous implementation of the `solution` parameter was either not clearing the cache or was not creating all appropriate connections (like having the solution also as parent), thus, making the distributions not available on the overview. 

I updated this test to check for the text to the header since this is a regression test after deciding to remove the licenses from the header region of the solution (previously they were shown in the little box where the download count also appears).